cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+Cypress.Commands.add('fillTable',
+  (fullName, position, city, age, salary) => {
```

Comment:
why fullName? why not name?

Reply:
name is already declared in the upper scope